server/zanata-frontend/src/frontend/app/styles/style.less

```diff
@@ -1905,7 +1905,7 @@ span.langcode {
 .btn-primary {
   color: #fff;
   border-color: #ccc;
-  background-color: @brand-primary;
```

Comment:
Should refactor @branch-primary to something else given its not really a branch-primary color now.